frontend/packages/dev-console/src/components/pipelineruns/PipelineTaskStatus.tsx

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable no-undef */
+import * as React from 'react';
+import { VictoryStack, VictoryBar } from 'victory';
+import { Tooltip } from '@console/internal/components/utils/tooltip';
+import { getTaskStatus, PipelineRun } from '../../utils/pipeline-augment';
+import '../pipelines/PipelineVisualizationTask.scss';
+
+interface PipelineTaskStatusProps {
+  pipelinerun: PipelineRun;
+}
+
+interface TaskStatusToolTipProps {
+  tooltip: Tooltip;
+}
+
+interface Tooltip {
```

Comment:
This should be the same type returned from `getTaskStatus` and it should have a name other than `Tooltip` if it's coming from a non-ui utility.